### DISCUSSION TITLE
Free daily sticker (Winter Sale 2022)

### DISCUSSION THREAD

**Discussion by pontifezo** [chr233/ASFEnhance#59] [功能建议 / Ideas]
Is it possible to get free daily sticker with ASFEnhance?

Stickers can be manually claimed during Winter Sale 2022 on games categories pages:

https://store.steampowered.com/greatondeck
https://store.steampowered.com/category/exploration_open_world
https://store.steampowered.com/category/rpg
...
etc

**Comment by woctezuma** [comment]
Already mentioned in:
- #57
